src/pages/apply/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { Container, Typography } from '@mui/material'
+import { Box } from '@mui/system'
+import { NextPage } from 'next'
+
+interface Props {}
+
+const BMApplicationFormPage: NextPage<Props> = ({}) => (
+  <Container>
+    <Box
+      sx={{
+        my: 4,
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        alignItems: 'center',
+      }}
+    >
+      <Typography variant="h6">BoilerMake Application page</Typography>
+      <p>
```

Comment:
You can also use `<Typography variant="body1"/>` instead of a `<p/>` tag. Semantically better

Reply:
I'm guessing we're removing this text anyways?